symplyphysics/laws/dynamics/deformation/shear_stress_is_shear_modulus_times_strain.py

```diff
@@ -0,0 +1,50 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## When an object is under a shearing stress, the shear stress applied to the object is proportional
+## to the shear modulus of the object and the ratio between the displacement of one end of the object
+## in the direction of the applied (shear) force and the length of the object.
```

Comment:
It is not really clear, what kind of deformation happens here. Is it bending? Or is it being pressed from one side towards another?
Maybe adding some picture would be handy.

Reply:
Yes.